package.json

```diff
@@ -52,7 +53,7 @@
     "@types/json-schema": "^7.0.11",
     "@types/react-dom": "^16.9.8",
     "@types/react-router-dom": "^5.3.2",
-    "cypress": "12.17.4",
+    "cypress": "^13.6.0",
```

Comment:
nit: honestly can prolly just delete as this is defined in osd package.json so this will be implicit